framework/doc/content/source/timeintegrators/NewmarkBeta.md

```diff
@@ -0,0 +1,29 @@
+# NewmarkBeta
+
+!syntax description /Executioner/TimeIntegrator/NewmarkBeta
+
+## Description
+
+Newmark time integration [citep:newmark1959amethod] is one of the commonly used time integration methods in structural dynamics problems. In this method, the second ($\ddot{u}$) and first (\dot{u}) time derivatives of a variable $u$ at $t+\Delta t$ are written in terms of the $u$, $\dot{u}$ and $\ddot{u}$ at time $t$, and $u$ at $t+\Delta t$ as shown below:
+
+\begin{equation}
+\begin{aligned}
+\mathbf{\ddot{u}}(t+\Delta t) &=& \frac{\mathbf{u}(t+\Delta t)-\mathbf{u}(t)}{\beta \Delta t^2}- \frac{\mathbf{\dot{u}}(t)}{\beta \Delta t}+\frac{\beta -0.5}{\beta}\mathbf{\ddot{u}}(t) \\
+\mathbf{\dot{u}}(t+ \Delta t) &=& \mathbf{\dot{u}}(t)+ (1-\gamma)\Delta t \mathbf{\ddot{u}}(t) + \gamma \Delta t \mathbf{\ddot{u}}(t+\Delta t)
+\end{aligned}
+\end{equation}
+
+In the above equations, $\beta$ and $\gamma$ are Newmark time integration parameters.
+
+- For $\beta = \frac{1}{4}$ and $\gamma = \frac{1}{2}$, the Newmark time integration method is implicit and unconditionally stable. This is the constant average acceleration method with no numerical damping. This is recommended only when a constant timestep is used throughout the simulation. If for some reason, the simulation does not converge and the timestep is halved, this time integration method with no numerical damping can result in high frequency noise.
+- $\beta = \frac{1}{6}$ and $\gamma = \frac{1}{2}$ results in the linear acceleration method where the acceleration is linearly varying between $t$ and $t+\Delta t$.
```

Comment:
If I understand correctly, the linear acceleration method is also second-order accurate, so what makes Newmark-beta superior? Is the linear accel. method *not* unconditionally stable? If it's not unconditionally stable, what are the timestep restrictions? If it is unconditionally stable, are there scenarios where it is not as good as Newmark?

Reply:
Linear acceleration method is a subset of Newmark-beta method. Linear acceleration method is also unconditionally stable and second order accurate. However, it adds some numerical damping which constant average acceleration method does not. 

This numerical damping can be beneficial in some scenarios. For e.g., in http://web.mit.edu/kjb/www/Principal_Publications/Insight_into_an_implicit_time_integration_scheme_for_structural_dynamics.pdf, it is shown that newmark-Beta time integration methods can cause high frequency noise in the velocity and acceleration for a displacement controlled problem. The oscillations are highest when there is no numerical damping, and by varying the newmark-beta parameters the magnitude of these oscillations can be reduced. However, the damping provided by newmark-beta methods is usually quite small and it does not completely remove the high frequency oscillations. The HHT-alpha method does a much better job of reducing high frequency noise.

Reply:
This is a great explanation, would you mind putting some version of this (including the link) into the documentation md file as well?

Reply:
I have added some more details about the different newmark-beta methods and HHT time integration. Please let me know if something else needs to be added.